zwave_js_server/model/controller/inclusion_and_provisioning.py

```diff
@@ -78,7 +78,6 @@ def from_dict(cls, data: Dict[str, Any]) -> "ProvisioningEntry":
         )
 
 
-@dataclass
```

Comment:
This should be a dataclass.

Reply:
it is, it's just in `controller.__init__`

Reply:
ah I accidentally removed the decorator from one of the classes, thanks for catching that 👍🏾